x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/index.ts

```diff
@@ -14,5 +14,6 @@ export function getAlertType(): AlertTypeModel {
     iconClass: 'alert',
     alertParamsExpression: IndexThresholdAlertTypeExpression,
     validate: validateExpression,
+    requiresAppContext: true,
```

Comment:
I think all the values of `requiresAppContext` need to be inverted now right? Since index threshold alert doesn't require app context so can be created / edited from management UIs.